casr/src/bin/casr-cluster.rs

```diff
@@ -63,37 +63,28 @@ fn make_clusters(
         bail!("{} reports, nothing to cluster...", len);
     }
 
-    casreps.sort_by(|a, b| {
-        a.file_name()
-            .unwrap()
-            .to_str()
-            .unwrap()
-            .cmp(b.file_name().unwrap().to_str().unwrap())
-    });
-
     // Start thread pool.
     let custom_pool = rayon::ThreadPoolBuilder::new()
         .num_threads(jobs.min(len))
         .build()
         .unwrap();
 
-    // Stacktraces from casreps
-    let traces: RwLock<Vec<Stacktrace>> = RwLock::new(Vec::new());
-    // Crashlines from casreps
-    let crashlines: RwLock<Vec<String>> = RwLock::new(Vec::new());
-    // Casreps with stacktraces, that we can parse
-    let filtered_casreps: RwLock<Vec<PathBuf>> = RwLock::new(Vec::new());
+    // Report info from casreps: (casrep, (trace, crashline))
```

Comment:
Maybe, just a flat tuple with 3 elements

Reply:
It's made according to `unzip()` documentation
https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.unzip

Reply:
For triple tuple get tree vectors is not such simple

Reply:
ok